api/query/query.go

```diff
@@ -61,6 +57,27 @@ func (qh queryHandler) processInput(w http.ResponseWriter, r *http.Request) (*sh
 	return &filters, testRuns, summaries, nil
 }
 
+func (qh queryHandler) validateSummaryVersions(v url.Values, logger shared.Logger) (bool, error) {
+	filters, err := shared.ParseQueryFilterParams(v)
+	if err != nil {
+		return false, err
+	}
+	testRuns, _, err := qh.getRunsAndFilters(filters)
+	if err != nil {
+		return false, err
+	}
+
+	for _, testRun := range testRuns {
+		summaryURL := shared.GetResultsURL(testRun, "")
+		// All new summary URLs end with "-summary_v2.json.gz".
+		if !strings.HasSuffix(summaryURL, "-summary_v2.json.gz") && !strings.HasSuffix(summaryURL, "-summary.json.gz") {
```

Comment:
In your change explanation, it says: `...ensures it ends with the -summary_v2.json.gz suffix`

`!strings.HasSuffix(summaryURL, "-summary_v2.json.gz") && !strings.HasSuffix(summaryURL, "-summary.json.gz")` will allow both v2 and v1.

Shouldn't v1 not be supported anymore?

Reply:
Yes, you're correct. I'm glad you caught this, as it's a leftover from the previous change that allows both but writes as `_v2` 😅. I'll fix this and write a test so that this would be caught in the future.